docs/editor/testing.md

```diff
@@ -0,0 +1,173 @@
+---
+Order: 8
+Area: editor
+TOCTitle: Testing
+ContentId: d44f1a5c-5454-4037-92d5-c2bf5d4cffed
+PageTitle: Testing in Visual Studio Code
+DateApproved: 05/02/2024
+MetaDescription: One of the great things in Visual Studio Code is testing support. Automatically discover tests in your project, run and debug your tests, and get test coverage results.
+---
+# Testing
+
+Software testing is a crucial aspect of the software development process. It helps ensure that your code works as expected and catches bugs early in the development cycle. Visual Studio Code provides a rich set of features for testing your code. You can automatically discover tests in your project, run and debug your tests, and get test coverage results.
+
+In this article, you'll learn how to get started with testing in VS Code, discover popular testing extensions, and explore the testing capabilities.
+
+![Testing in Visual Studio Code](images/testing/testing-hero.png)
+
+## Getting started with testing in VS Code
+
+Testing support in VS Code is language-specific and depends on the [extensions](#testing-extensions) you have installed. The testing features are provided by the language extensions, which can be language servers, test runners, or other tools that support testing in that language. To get started with testing in VS Code, first install the appropriate language extension for your project.
```

Comment:
Testing can be in language extensions, but not always. E.g. there's no de-facto standard test framework for JavaScript so support is found in standalone extensions.